sw/device/tock/README

```diff
@@ -0,0 +1,43 @@
+Tock OS
```

Comment:
Please move to README.md so this renders nicely in the GitHub UI.

We tend to use the more common `# Heading`, `## Subheading` syntax for headings in Markdown too

Reply:
done.